engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/artificial/AliasGenerator.kt

```diff
@@ -0,0 +1,41 @@
+package graphql.nadel.enginekt.transform.artificial
+
+import graphql.normalized.ExecutableNormalizedField
+import java.util.UUID
+
+object AliasGenerator {
+    /**
+     * Random substrings to attach to the field alias to make it unique enough such
+     * that nobody should ever guess the field result key and mess with the fields
+     * introduced by Nadel.
+     */
+    private val suffixes = Array(100) {
+        UUID.randomUUID().toString().substringAfterLast("-")
```

Comment:
Decided to cut it short because we don't need a full UUID in the request and response bloating the size.